services/gitlab/gitlab-issues.service.js

```diff
@@ -0,0 +1,446 @@
+import Joi from 'joi'
+import { optionalUrl } from '../validators.js'
+import { metric } from '../text-formatters.js'
+import GitLabBase from './gitlab-base.js'
+
+const schema = Joi.object({
+  statistics: Joi.object({
+    counts: Joi.object({
+      all: Joi.number().required(),
+      closed: Joi.number().required(),
+      opened: Joi.number().required(),
+    }).required(),
+  }).allow(null),
+}).required()
+
+const queryParamSchema = Joi.object({
+  labels: Joi.string(),
+  gitlab_url: optionalUrl,
+}).required()
+
+const documentation = `
+<p>
+  You may use your GitLab Project Id (e.g. 278964) or your Project Path (e.g. gitlab-org/gitlab )
+</p>
+`
+
+const customDocumentation = `
+<p>
+  Note that only internet-accessible GitLab instances are supported, for example https://jihulab.com, https://gitlab.gnome.org, or https://gitlab.com/.
+  You may use your GitLab Project Id (e.g. 13953) or your Project Path (e.g. gitlab-cn/gitlab ) in <a href="https://jihulab.com">https://jihulab.com</a>
+</p>
+`
+
+const labelDocumentation = `
+<p>
+  Support Multi-label and <a href="https://docs.gitlab.com/ee/user/project/labels.html#scoped-labels">Scoped labels</a>, Multi-lable please use <code>,</code> to separate.
+</p>
+`
+
+export default class GitlabIssues extends GitLabBase {
+  static category = 'issue-tracking'
+
+  static route = {
+    base: 'gitlab/v/issues',
+    pattern: ':variant(all|opened|closed):raw(-raw)?/:project+',
```

Comment:
We don't have to make our user-facing naming conventions directly map to the underlying fields in the upstream responses, and more often than not, we don't want to.

In this case, I believe `open` would be a better variant name, and would also eliminate some of the re-mapping you are currently performing in the rendering function.

However, I am operating under the assumption that the GitLab issue specifics are showing the number of issues currently open (optionally with the specified labels applied), as opposed to every issue that was ever opened, including those which may be close. I make that assumption because if GitLab didn't, then I feel like the `all` and `opened` fields in the API response would be synonymous, and duplicative. As such the GitLab field name feels like a bit of a confusing misnomer, and we should correct that in our route variant.

Reply:
done